api/src/api/elevatorSafetyRouter.ts

```diff
@@ -18,5 +22,18 @@ export const elevatorSafetyRouterFactory = (
       });
   });
 
+  router.post("/elevator-safety/registration/", async (req, res) => {
+    const { address } = req.body;
```

Comment:
dust, we should extract address and zip code and remove the `const zipCode` in the next line.

Can the log be removed?

We are avoiding using the `as` keyword to override the type, is there another way around this?